### PR TITLE
[console-bridge] Update to 1.0.2

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates console-bridge to 1.0.2

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run ./vcpkg x-add-version --all and committed the result?  
  Yes

### CODE REVIEW

File: ports/console-bridge/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,17 @@
 {
   "name": "console-bridge",
-  "version-string": "1.0.1",
-  "port-version": 2,
-  "description": "a ROS-independent package for logging that seamlessly pipes into rosconsole/rosout for ROS-dependent packages.",
-  "homepage": "https://github.com/ros/console_bridge"
+  "version-semver": "1.0.2",
```
Comment (chausner) [inline ports/console-bridge/vcpkg.json:3]: Semver according to https://github.com/ros/console_bridge/blob/9a713ca3832df631507b1300b4f2d66747e48534/QUALITY_DECLARATION.md#version-scheme-1i.

File: ports/console-bridge/portfile.cmake [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-file(READ ${SOURCE_PATH}/src/console.cpp _contents)
+file(READ "${SOURCE_PATH}/src/console.cpp" _contents)
 string(SUBSTRING "${_contents}" 0 2000 license)
-file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/share/console-bridge)
-file(WRITE ${CURRENT_PACKAGES_DIR}/share/console-bridge/copyright "${license}")
+file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share/console-bridge")
+file(WRITE "${CURRENT_PACKAGES_DIR}/share/console-bridge/copyright" "${license}")
```
Comment (Thomas1664) [inline ports/console-bridge/portfile.cmake:33]: They now have their own license file as well. Please put this at the end of the script.

**Review by Thomas1664 — CHANGES_REQUESTED** [review]

File: ports/console-bridge/portfile.cmake [hunk 3]
```diff
@@ -1,42 +1,41 @@
...
+    REF 1.0.2
+    SHA512 ed427da8e59f9629f8d70e0a14415f88177c06fbaf7334bee56135dde91d19a1b54f5c9c668e0fd68314ab8dfd61446a174b9f528304decc5d4626a7c98882cb
     HEAD_REF master
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
```
Comment (Thomas1664) [inline ports/console-bridge/portfile.cmake:10]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS
        -DBUILD_TESTING=OFF
```

**Review by Thomas1664 — CHANGES_REQUESTED** [review]

File: ports/console-bridge/portfile.cmake (hunk 2 above)
Comment (chausner) [reply ports/console-bridge/portfile.cmake]: Good catch, done in b9ff464ffb5f5cccc7f476e7e86ca8ba68e5ce0b.

File: ports/console-bridge/portfile.cmake (hunk 3 above)
Comment (chausner) [reply ports/console-bridge/portfile.cmake:10]: Done in b9ff464ffb5f5cccc7f476e7e86ca8ba68e5ce0b.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**